srcpkgs/QSyncthingTray/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'QSyncthingTray'
+pkgname=QSyncthingTray
+version=0.5.8
+revision=1
+build_style=cmake
+makedepends="qt5-declarative-devel qt5-webengine-devel qt5-webchannel-devel qt5-location-devel qt5-quickcontrols2-devel qt5-devel"
+short_desc="A OS X / Windows / Linux Tray App for Syncthing written in C++"
+maintainer="Morgan Bazalgette <the@howl.moe>"
+license="LGPL-3.0"
```

Comment:
Pls use [SPDX Identifier ](https://spdx.org/licenses/)

Reply:
There is no clear indication whether the license is LGPL 3.0 only or LGPL 3.0 or later - should I assume it means "or later"?

Reply:
You have to take a look at the source files to determine it.
https://github.com/sieren/QSyncthingTray/blob/master/sources/qst/statswidget.cpp
The header there states `version 3.0 of the License, or (at your option) any later version.`